src/Lean/Meta/WHNF.lean

```diff
@@ -63,19 +63,12 @@ def isAuxDef (constName : Name) : MetaM Bool := do
   let env ← getEnv
   return isAuxRecursor env constName || isNoConfusion env constName
 
-@[inline] private def matchConstAux {α} (e : Expr) (failK : Unit → MetaM α) (k : ConstantInfo → List Level → MetaM α) : MetaM α := do
-  let .const name lvls := e
-    | failK ()
-  let (some cinfo) ← getUnfoldableConst? name
-    | failK ()
-  k cinfo lvls
```

Comment:
I found the code clearer after inlining this helper, its name is too mysterious otherwise